src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -1,367 +0,0 @@
-## 1.16.0a1 (tbd)
```

Comment:
Don't mind removing this file, however - can we fold for older versions (PrPr) the Pandas versions into the changelog above? I.e., something like this:

```

## 1.15.0a1 (Snowpark pandas API PrPr private release)
...
## 1.15.0
....
```

For the public releases, let's have an explicit announcement in the Changelog that pandas API is now part of Snowpark, and then have the changes be part of the changelog as you propose with this PR.